lib/output.js

```diff
@@ -53,7 +55,16 @@ const labData = lighthouseResult => {
   return sortOn(ret, 'label');
 };
 
-const opportunities = lighthouseResult => {
+const getTitle = (title, description) => {
+  let output = title;
+  if (description) {
```

Comment:
I am wondering a bit about when getLink is `''`, and what that does for terminalLink.

Does it make sense to catch this kind of error-ish result from getLink.

I may be getting to `golang` with this since that's what I've been doing lately haha.

```javascript
if (description) {
  let link = getLink(description);
  if (!link) return output
  output = terminalLink(title, link);
}
```

Reply:
Or even flatter.

```javascript
const getTitle = (title, description) => {
  if (!description) return title;

  let link = getLink(description);
  if (!link) return title;

  return terminalLink(title, link);
}
```

I might be going too far 😆

Reply:
Yep, you are right. `getTitle` might handle the correct output checking `description` and `link` truthiness. 

I like your proposals a lot, but I think that it's better to use  positive assertions instead and only have one return per function. But I suppose that is a matter of taste or 'company code style'.

My proposal, simpler:

```javascript
const getTitle = (title, description) => {
  let output = title;
  if (description) {
    const link = getLink(description);
    if (link) {
      output = terminalLink(title, getLink(description));
    }
  }

  return output;
};
```

Thoughts?

Reply:
Yours looks fine 😄

Reply:
Thanks 😄 

Change done.